Sources/Plasma/FeatureLib/pfPasswordStore/pfPasswordStore_impl.h

```diff
@@ -75,10 +75,19 @@ class pfWin32PasswordStore : public pfPasswordStore
 };
 #endif //HS_BUILD_FOR_WIN32
 
-
+#if defined(HS_BUILD_FOR_UNIX) && !defined(HS_BUILD_FOR_OSX)
 /**
-* @todo A Linux libsecret-based storage mechanism.
-*/
+ * A libsecret-based password storage mechanism.
+ */
+class pfUnixPasswordStore : public pfPasswordStore
```

Comment:
You need to update pfPasswordStore.cpp to actually create a pfUnixPasswordStore when we ask for an instance